src/service/app/statistics.ts

```diff
@@ -20,9 +20,13 @@ export const getStatistics = (dispatch: AppDispatch) => {
       if (isMainnet) {
```

Comment:
Should we add a condition if `fetchCachedData(AppCachedKeys.HardForkInfo)?.hasFinishedHardFork` is false?

Reply:
Updated in https://github.com/nervosnetwork/ckb-explorer-frontend/pull/936/commits/40d0e47ab41bccf289397e22e1a2ccee2b2e5c8c